.editorconfig

```diff
@@ -46,6 +46,9 @@ dotnet_style_collection_initializer = true:error
 dotnet_style_coalesce_expression = true:error
 dotnet_style_null_propagation = true:error
 dotnet_style_explicit_tuple_names = true:error
+dotnet_style_allow_multiple_blank_lines_experimental=false:error

```

Comment:
Is een experimentele setting wel handig om als default te setten?

Reply:
Ik ben voorstander van deze feature, maar wat is de compatibility? (als het niet herkent wordt breekt het iets of wordt het gewoon genegeerd)

Reply:
Wordt genegeerd. 

Misschien moet iemand ff naar `https://github.com/dotnet/docs/issues/28791` kijken om te zien of we nog meer missen.